app/src/main/java/org/qosp/notes/ui/notebooks/ManageNotebooksViewModel.kt

```diff
@@ -4,16 +4,36 @@ import androidx.lifecycle.ViewModel
 import androidx.lifecycle.viewModelScope
 import dagger.hilt.android.lifecycle.HiltViewModel
 import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.async
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.flow.map
 import kotlinx.coroutines.launch
+import kotlinx.coroutines.runBlocking
 import org.qosp.notes.data.model.Notebook
 import org.qosp.notes.data.repo.NotebookRepository
+import org.qosp.notes.preferences.PreferenceRepository
 import javax.inject.Inject
 
 @HiltViewModel
-class ManageNotebooksViewModel @Inject constructor(private val notebookRepository: NotebookRepository) : ViewModel() {
+class ManageNotebooksViewModel @Inject constructor(
+    private val notebookRepository: NotebookRepository,
+    private val preferenceRepository: PreferenceRepository
+) : ViewModel() {
     fun deleteNotebooks(vararg notebooks: Notebook) {
         viewModelScope.launch(Dispatchers.IO) {
             notebookRepository.delete(*notebooks)
         }
     }
+
+    fun getSortNavdrawerNotebooksMethod() : String {
+        return runBlocking {
+            return@runBlocking async {
+                preferenceRepository
+                    .getAll()
+                    .map { it.sortNavdrawerNotebooksMethod }
+                    .first()
+                    .name
+            }.await()
```

Comment:
Same comment as earlier

Reply:
Done. Carried out in f949e9c4118a72110ecfe55d7de3fbf63caaedbf.